bitnami/nats/templates/ingress.yaml

```diff
@@ -1,6 +1,11 @@
 {{- if .Values.ingress.enabled -}}
+{{- $apiVersions := .Capabilities.APIVersions -}}
 {{- range .Values.ingress.hosts }}
+{{- if $apiVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
In other charts we use a helper function that returns the proper API version.

```
apiVersion: {{ include "nats.ingressAPIVersion" . }}
```

Reply:
Thanks your review, and fix it!